src/commtrack.js

```diff
@@ -136,7 +136,7 @@ define([
                 return {
                     xmlns: LEDGER_XMLNS,
                     type: mug.p.nodeID,
-                    "entity-id": attrs.src || "",
+                    "entity-id": attrs["entity-id"] || "",
```

Comment:
This was a (probably benign) bug in the commtrack module. Unrelated to copy/paste.